docs/en_US/Tutorial/NniOnWindows.md

```diff
@@ -1,6 +1,8 @@
-# NNI on Windows (experimental feature)
+# NNI on Windows
+Differ from Linux version, NNI only supports local mode right now. Please comment in [NNI's backlog issue](https://github.com/microsoft/nni/issues/1917) if you have needs on other modes for NNI on Windows.
```

Comment:
... right now. For other modes, you could use Windows as a machine to start an NNI experiment. For example, in remote mode, you could use Windows to run `nnictl` commands and it connects several Linux machines for running trials. Similarly, you could start an NNI experiment on Windows to run trials in PAI mode. But note that Windows cannot act as a remote server for running trials.

Reply:
> But note that Windows cannot act as a remote server for running trials.
we might need one diagram to explain nni's runtime nodes, like: nnictl node + trials node(s).

Reply:
agree

Reply:
Updated, @xuehui1991, @leckie-chn , @QuanluZhang - please help to review.